From: SJ Park <sj@kernel.org>
Date: Sat, 24 Oct 2026 12:25:31 -0700
Subject: [PATCH] ==== three race fixes ====

mm/damon: fix three race bugs in sysfs and core

There are three race condition bugs in DAMON sysfs interface and the
core layer.  User impacts are not critical but better to fix sooner than
later.

Patch 1 fixes a toctou bug in DAMON sysfs interface that can mess up
DAMOS auto-tuning.  Patches 2 and 3 move static variables that can be
wrongly shared with multiple kdamonds and therefore make DAMON
monitoring results be a little more indeterministic.

Note for Andrew.  These are not urgent but better to take the hotfix
track in my humble opinion.

Signed-off-by: SJ Park <sj@kernel.org>
---
 hkml_cv_bogus/hkml_cv_bogus_jxa64w1g | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 hkml_cv_bogus/hkml_cv_bogus_jxa64w1g

diff --git a/hkml_cv_bogus/hkml_cv_bogus_jxa64w1g b/hkml_cv_bogus/hkml_cv_bogus_jxa64w1g
new file mode 100644
index 0000000000000..e69de29bb2d1d
-- 
2.47.3

